docs/content/topics/contributor-guide/reviewing.md

```diff
@@ -0,0 +1,121 @@
+and _how_.
+
+> ⚠️&nbsp;&nbsp;Throughout this document, "project" (with a lowercase "p") is
+> used to denote individual software projects or source code repositories that
+> are related in some way to Brigade and owned by the
+> [@brigadecore](https://github.com/brigadecore) GitHub org. In contrast,
+> "Project" (with an uppercase "P") is used to denote the entire breadth of
+> Brigade as a "program" encompassing Brigade itself and _all_ related software
+> projects or source code repositories owned by the __@brigadecore__ org.
+
+## Reviewers
+
+For the Brigade Project, __Reviewer__ is not a formal role in the way that a
+__Project Maintainer__ or __Core Maintainer__ is. Here, a Reviewer is anyone
+who reviews a PR, whether voluntarily or to fulfill commitments they have to the
+Project. This is to say, feedback on any change is always welcome from _any_
+Community Participant who adheres to the
+[CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
+
+Despite reviews and comments being welcome from all Community Participants,
+those reviews and comments carry varying degrees of weight under different
+circumstances.
+
+### PRs Opened by Contributors
+
+PRs opened by Contributors who are not also a Core Maintainer or a Project
+Maintainer of the project in question must be reviewed and approved _by_ a Core
+Maintainer or a Project Maintainer. Feedback is welcome from all Community
+Participants, but ultimate approval or rejection of the PR is at the discretion
+of a Core Maintainer or Project Maintainer.
+
+### PRs Opened by Maintainers
+
+PRs opened by Project Maintainers and Core Maintainers fall into two categories:
+
+* __Ordinary PRs:__ These require a thorough review from a peer Core Maintainer
+  or Project Maintainer, the same as if they'd been opened by any other
+  Contributor. Feedback is welcome from all Community Participants, but ultimate
+  approval or rejection of the PR is at the discretion of a Core Maintainer or
+  Project Maintainer _who is not the author of the PR_.
+
+* __Chore PRs:__ To prevent the relatively small number of Core Maintainers and
+  Project Maintainers from becoming review bottlenecks, a Core Maintainer or
+  Project Maintainer (if opening a PR for their own project) may use their own
+  discretion to classify their PR as a "chore" by applying the "chore" label to
+  the PR. This indicates that the PR is (subjectively) small in scope and is
+  likely be understood by any Community Participant -- even those _without_ deep
+  technical knowledge of Brigade. With this designation, _any_ Community
+  Participant may review and approve the PR.
+
+## When to Review a PR
+
+If a PR is in draft form, review/comments are permissible, but assume that the
+author is not explicitly _requesting_ any sort of review or approval.
+Contributors often open draft PRs for "safe keeping" or just to create some
+visibility into their in-progress work. Assume the author of a draft _knows_
```

Comment:
I like calling this out, "Yes, Bob I realize that the PR isn't finished, thanks for the review" 🤣